config/jobs/kubernetes-csi/external-resizer/external-resizer-config.yaml

```diff
@@ -12,10 +12,6 @@ presubmits:
       preset-service-account: "true"
       preset-dind-enabled: "true"
       preset-kind-volume-mounts: "true"
-    annotations:
```

Comment:
we should replace this with updated testgrid annotations

Reply:
The old repos were whitelisted as being allowed to have no matching testgrid entry. I think we can remove the [whitelist](https://github.com/msau42/test-infra/blob/00807fd2ba8a72631faa445bdc5c076748875a0c/config/tests/testgrids/config_test.go#L341) now

Reply:
I agree on removing the whitelist. But that won't fix the verification error, will it?